README.md

```diff
@@ -1,20 +1,21 @@
 [![Build Status](https://travis-ci.org/ModellingWebLab/cellmlmanip.svg?branch=develop)](https://travis-ci.org/ModellingWebLab/cellmlmanip) [![Documentation Status](https://readthedocs.org/projects/cellmlmanip/badge/?version=latest)](https://cellmlmanip.readthedocs.io/en/latest/?badge=latest) [![Coverage Status](https://codecov.io/gh/ModellingWebLab/cellmlmanip/branch/develop/graph/badge.svg)](https://codecov.io/gh/ModellingWebLab/cellmlmanip/)
 
 # cellmlmanip
-CellML loading and model equation manipulation
+CellML loading and model equation manipulation library
 
-## Developer installation
+## Installing 
+We reccomend installing chaste_codegen in a vritual environment (using conda or virtualenv)
 
-At the moment, there is no user installation procedure for cellmlmanip.
+Users install `cellmlmanip` using pip.
```

Comment:
Probably just say "Cellmlmanip can be installed using pip:"

No need to differentiate between user/devel here. Also not sure it's called `cellmlmanip`, in code style, is it?

Reply:
do you have a better idea for naming the pypi pacakge?

Reply:
OK, I understand what you're saying now :D

Reply:
I would write something like "Users would install Cellmlmanip using `pip install cellmlmanip`"